react-ui/src/pages/challenge.page.jsx

```diff
@@ -8,6 +8,7 @@ import LikeAndFollowChallenge from '../components/Challenges/like-and-follow-cha
 import ShowOffChallenge from '../components/Challenges/show-off-challenge';
 import twitterIcon from '../assets/twitter-icon.svg';
 import SocialIcons from '../components/SocialIcons/social-icons';
+import SelfMonitor from '../components/Challenges/self-monitor
```

Comment:
This line is missing a single quote and semi-colon at the end of it and will throw an error when you compile the app.